day3/part1.py

```diff
@@ -0,0 +1,37 @@
+if __name__ == "__main__":
+    
+    # Read text file into numbers within an array
+    with open("day3/input.txt") as values:
+         binary_input = values.readlines()
+
+    # Why is len(binary_input[0]) = 6?
+    col = len(binary_input[0]) - 1   

```

Comment:
You should check and see what the last character is in this string, I'm sure you can run this in the debugger and find out why that's going on

Reply:
It says the binary number is 13 digits long. When I print the 13th digit, nothing shows up as being printed